Sources/AppStoreConnectClient/AppStoreConnectClient.swift

```diff
@@ -60,40 +60,92 @@ public struct AppStoreConnectClient {
                 return json.data.compactMap({ Application(schema: $0) })
             }
         case .badRequest(let result):
-            throw AppStoreConnectError.badRequest
+            switch result.body {
```

Comment:
I believe that we can use 'result.body' as param for 'handleError'. In this case we will be able to omit  duplications of switch/cases

Reply:
Changed.